staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -740,6 +741,9 @@ func waitForContainer(ctx context.Context, podClient corev1client.PodsGetter, ns
 			if s.State.Running != nil || s.State.Terminated != nil {
 				return true, nil
 			}
+			if !o.Quiet && s.State.Waiting != nil && s.State.Waiting.Message != "" {
+				fmt.Fprintf(o.ErrOut, "Container %s: %s\n", containerName, s.State.Waiting.Message)
```

Comment:
To make this easier to find later for https://github.com/kubernetes/kubernetes/pull/108210
```suggestion
				fmt.Fprintf(o.ErrOut, "Warning: container %s: %s\n", containerName, s.State.Waiting.Message)
```